bootstrap-salt.sh

```diff
@@ -3671,8 +3759,12 @@ install_centos_git_deps() {
         __PACKAGES="${__PACKAGES} python-libcloud"
     fi
 
-    # shellcheck disable=SC2086
-    __yum_install_noinput ${__PACKAGES} || return 1
+    if [ "${_INSTALL_PY}" = "${BS_TRUE}" ]; then
+        __install_python_and_deps || return 1
+    else
+        # shellcheck disable=SC2086
+        __yum_install_noinput ${__PACKAGES} || return 1
```

Comment:
Is this supposed to be in the `else` statement here? It seems like we've done a lot of work up to this point to add many `python-*` packages to the `__PACKAGES` list, and now they won't be installed if you're installing a new version of Python. Are we passing that `__PACKAGES` chain to `__install_python_and_deps` function that I am not seeing?